src/state/authen/reducer.ts

```diff
@@ -0,0 +1,40 @@
+import { createReducer } from '@reduxjs/toolkit'
+
+import { updatePossibleWalletAddress, updateProcessingLogin, updateProfile } from './actions'
+
+export type UserProfile = { email: string; identityId: string; data: { hasAccessToKyberAI: boolean } }
+export interface AuthenState {
+  readonly possibleConnectedWalletAddress: null | string | undefined // null is checking
```

Comment:
what does it means `null is checking`

Reply:
useActiveWeb3 react slow return `account`. (although we connected wallet but it return undefined => wait a bit => account = 0x....)
I need it to early get wallet address, or detect connecting/activating wallet.
When you upgrate to web3-react, we can remove it. 
anw, I updated description of this state in the code

Reply:
Please help me provide all case of these your wallet update, I will note it and check in v8 update @nguyenhoaidanh